src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs

```diff
@@ -225,8 +238,11 @@ void Connect()
 
                 _eventSource.Changed += OnEventSourceChanged;
 
-                if (_dataSource.TextChangeBehavior.HasFlag(TaggerTextChangeBehavior.TrackTextChanges))
+                if (_dataSource.TextChangeBehavior.HasFlag(TaggerTextChangeBehavior.RemoveAllTags) ||
```

Comment:
>_dataSource.TextChangeBehavior [](http://example.com/codeflow?start=20&length=30)

maybe 

_dataSource.TextChangeBehavior != TaggerTextChangeBehavior.None

Reply:
can change in followup.